src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Samsung</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Smartphones</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#"
+              class="navigation__link"
+              data-qa="nav-hover">
+            Laptops & Computers
+            </a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Gadgets</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Tablets</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Photo</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card card--location" data-qa="card">
+        <img src="images/imac.jpeg"
+          alt="imac"
+          class="card__photo">
+
+        <h3 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="card__stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
```

Comment:
add some space to improve readability

Make the same to each card
```suggestion
          </div>
          
          <p class="card__reviews">Reviews: 5</p>
```